lib/releaselib.sh

```diff
@@ -896,7 +896,7 @@ release::docker::release () {
   local version=$2
   local build_output=$3
   local release_images=$build_output/release-images
-  local docker_push_cmd=(docker)
+  local -a docker_push_cmd=("$GCLOUD" "docker" "--")
```

Comment:
`gcloud docker --` is no longer supported AFAIK - we should be using only `docker`.